test/inductor/test_fx_fusion.py

```diff
@@ -30,7 +30,7 @@ def parent_pass(module: torch.fx.GraphModule, input: Any) -> torch.fx.GraphModul
 
 def count_call(module: torch.fx.GraphModule, op: str, target_op: Any) -> int:
     return sum(
-        [1 if (n.op == op and n.target == target_op) else 0 for n in module.graph.nodes]
+        1 if (n.op == op and n.target == target_op) else 0 for n in module.graph.nodes
```

Comment:
We do `sum` over the list of bools all over the place, why not use the same pattern here
```suggestion
        n.op == op and n.target == target_op for n in module.graph.nodes
```